mappings/net/minecraft/entity/passive/HorseEntity.mapping

```diff
@@ -16,3 +16,5 @@ CLASS net/minecraft/class_1498 net/minecraft/entity/passive/HorseEntity
 		ARG 1 stack
 	CLASS class_1499 HorseData
 		FIELD field_6994 variant Lnet/minecraft/class_5149;
+		METHOD <init> (Lnet/minecraft/class_5149;)V
+			ARG 1 color
```

Comment:
Would be better if the field and the parameter matched

Reply:
Looks like the parameter was right and the field was wrong.